docs/en/stack/ml/anomaly-detection/ml-troubleshooting.asciidoc

```diff
@@ -8,6 +8,42 @@
 Use the information in this section to troubleshoot common problems and known
 issues.
 
+[discrete]
+[[ml-avoid-upgrade-closures]]
+== Avoiding spurious {anomaly-job} closures on upgrade
+
+When you perform a rolling upgrade to _or_ from versions
```

Comment:
I think it would be helpful to point folks to the instructions for rolling upgrades:
```suggestion
When you perform a {ref}/rolling-upgrades.html[rolling upgrade] to _or_ from versions
```